src/main/java/me/mrCookieSlime/Slimefun/api/BlockStorage.java

```diff
@@ -97,6 +97,10 @@ private static Location deserializeLocation(String l) {
     public BlockStorage(World w) {
         this.world = w;
 
+        if (world.getName().indexOf('.') != -1) {
```

Comment:
I would also check a space. Spaces can mess up quite a bit on different systems. We've had someone in the past who downloaded a world and it messed up on the shared Linux server due to that

Reply:
Never heard of that happening. If someone reports it, it can be added. Otherwise, feel free to make a PR. This is just a dirty band aid for a much deeper problem anyway.